package/lib/index.js

```diff
@@ -650,17 +655,33 @@ global.Set.prototype.intersect = arbitraryToBinary(intersectionArbitrary)
 function difference (set1, set2) {
   checkSet(set1)
   checkSet(set2)
-  const set3 = new Set([])
+  const set3 = new Set()
   set1.forEach(value => {
     if (!set2.has(value)) {
       set3.add(value)
     }
   })
   return set3
 }
-
 global.Set.difference = difference
 
+/**
+ * Computes the set difference of two sets (subtracts B from A). This is also known as the "relative complement".
+ * <br>Expression: <code>A \ B = S</code>
+ * <br>Example: <code>{1,2,3} \ {1,3,5} = {2}</code>
+ * @example
+ * const A = Set.from(1, 2, 3)
+ * const B = Set.from(1, 3, 5)
+ * A.minus(B) // Set { 2 }
+ * @name Set.prototype.minus
```

Comment:
I was thinking about the `minus` and the [ link you showed me once](https://doc.sagemath.org/html/en/reference/sets/sage/sets/set.html#sage.sets.set.Set_object.difference), which is for example using `A.difference(B)`. Do you think it makes sense to keep up with their naming convention, too?

Reply:
Well it's a question of what sort of naming convention you want to follow.

  * My original proposed naming convention (convention 1) is about typing what you would say or write.  So "the difference of A and B" is our `Set.difference(A, B)` while "A minus B" is our `A.minus(B)`.

  * An alternative convention (convention 2) would be to have all binary operators use the *same name* as their corresponding arbitrary operator.  So having `Set.union(A, B)` means that we should have `A.union(B)` and similarly, having `Set.difference(A, B)` means that we should have `A.difference(B)`.

There's a few pros and cons to each convention, but I think they are both good choices.  Would you like to go with convention 2?  (Also, i think you are right that it is good to be consistent with Sagemath, with the exception of scenarios where they have made a bad choice)

Reply:
With convention 2, we would also be using `A.intersection(B)` instead of `A.intersect(B)`, which again matches with Sagemath's choice (http://doc.sagemath.org/html/en/reference/sets/sage/sets/set.html#sage.sets.set.Set_object.intersection).

Reply:
Okay I see your point now. Using the main term for the arbitrary operation on the `Set` class while using a more spoken / active term for the binary operations on the instance. I think we will have to figure this out while using the package in the long term. I will keep this for now but start a new Major release as the changes are breaking and deliver a whole new API.